components/config/src/config/link_checker.rs

```diff
@@ -7,10 +7,15 @@ pub struct LinkChecker {
     pub skip_prefixes: Vec<String>,
     /// Skip anchor checking for these URL prefixes
     pub skip_anchor_prefixes: Vec<String>,
+    pub nb_threads_per_cpu: usize,
```

Comment:
I don't think this should be configurable. Let's keep it at 4 for now and we'll see for configuration later on.

Reply:
So I come back to the static version